plugin.py

```diff
@@ -31,6 +41,15 @@ def arch() -> str:
         raise RuntimeError("Unknown architecture: " + sublime.arch())
 
 
+def get_setting(view: sublime.View, key: str, default: Optional[Union[str, bool]] = None) -> Union[bool, str]:
+    if view:
+        settings = view.settings()
+        if settings.has(key):
+            return settings.get(key)
+    settings = sublime.load_settings('LSP-rust-analyzer.sublime-settings')
+    return settings.get(key, default)
```

Comment:
Would be good to add those settings that you check for here to the schema and the default configuration file.

Reply:
Confused by what you mean here

Reply:
The `terminus_use_panel` and `terminus_auto_close` options are not documented, nor included in the default settings nor in the schema.

Reply:
I would add one more issue here.

The setting names `terminus_use_panel` and `terminus_auto_close` are very generic, and getting them from the `View` implies that those can be set in global Preferences.sublime-settings. That doesn't seem ideal because this setting is very specific to this package (I believe?) so in the context of the global preferences those would be very confusing. I think you should prefix them with `rust-analyzer.*` or something.

Reply:
Okay thats Fair I'll make a setting for them